src/Reflection/TypeReflection/TypeConverter.php

```diff
@@ -22,9 +22,13 @@ public function string(Type $self): mixed
         return NamedTypeReflection::string();
     }
 
-    public function int(Type $self): mixed
+    public function intRange(Type $self, ?int $min, ?int $max): mixed
```

Comment:
We can rename this method to `int` in `0.4.0`, @klimick, WDYT?

Reply:
I like this idea